ide/coqide.ml

```diff
@@ -439,7 +439,9 @@ let compile sn =
   match sn.fileops#filename with
     |None -> flash_info "Active buffer has no name"
     |Some f ->
+      let (_,args) = make_coqtop_args fn in
```

Comment:
For the sake of internal consistency, it might be cleaner to use `Coq.get_arguments sn.coqtop`.